SiraUtil/Sabers/Effects/SaberBurnMarkAreaLatch.cs

```diff
@@ -107,18 +95,5 @@ internal void BurnAreaStarting(SaberBurnMarkArea __instance)
                 AddSaber(siraSaber.Saber);
             _earlySabers.Clear();
         }
-
-        [AffinityPostfix]
-        [AffinityPatch(typeof(SaberBurnMarkArea), nameof(SaberBurnMarkArea.LateUpdate))]
-        internal void AbsoluteCarouselRenderShift(ref RenderTexture[] ____renderTextures)
-        {
-            // Shift every render texture down one in the array
-            RenderTexture lastTexture = ____renderTextures[____renderTextures.Length - 1];
-            for (int i = ____renderTextures.Length - 1; i > 0; i--)
-            {
-                ____renderTextures[i] = ____renderTextures[i - 1];
-            }
-            ____renderTextures[0] = lastTexture;
-        }
```

Comment:
This was breaking burn marks since the Graphics.Blit & texture swapping is no longer done every frame. I don't think this was actually doing anything useful anyway since SiraUtil isn't adding its own textures? Anyway, removing this fixes 1.36.0 and doesn't break 1.35.0.